### PR TITLE
counter-script docs

### PR DESCRIPTION
**Pull request by SwayStar123**
## Type of change

- Documentation

## Changes

Added documentation according to the src-2 standard

### CODE REVIEW

File: counter-script/project/scripts/interaction_script/src/main.sw [hunk 1]
```diff
@@ -2,6 +2,16 @@ script;
 
 use libraries::Counter;
 
+/// Calls the methods of the counter contract at the given contract_id.
```
Comment (Braqzen) [inline counter-script/project/scripts/interaction_script/src/main.sw:5]:
```suggestion
/// Calls the functions of the counter contract at the given contract_id.
```

File: counter-script/project/contracts/counter/src/main.sw [hunk 2]
```diff
@@ -3,6 +3,7 @@ contract;
 use libraries::Counter;
 
 storage {
+    /// The internal count to be manipulated.
```
Comment (Braqzen) [inline counter-script/project/contracts/counter/src/main.sw:6]:
```suggestion
    /// Internal counter updated via calls from a script.
```

**Review by Braqzen — APPROVED** [review]

File: counter-script/project/scripts/interaction_script/src/main.sw (hunk 1 above)
Comment (SwayStar123) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: Why?

Comment (Braqzen) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: https://fuellabs.github.io/sway/master/reference/documentation/language/functions/index.html

Comment (SwayStar123) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: https://fuellabs.github.io/sway/master/reference/documentation/examples/counter.html?highlight=Contract%20method#implementation

Comment (Braqzen) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: d84b3210af89d64439c07e7c7b3c073367b834c65c4ed76f4c72a77c87a011d9_1-457759858

Yh, that should be fixed too

Comment (SwayStar123) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: No, the term methods is correct. Even in the rust sdk you do .methods on the contract object to access the methods remember?

Comment (Braqzen) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: It's an associated function. For UX you're going to call it a `.method()` in the SDK.

Comment (K1-R1) [reply counter-script/project/scripts/interaction_script/src/main.sw:5]: Moving ahead with method, as per internal discussion

**Review by K1-R1 — APPROVED** [review]

**Pull request merged**